deploy/modules/__init__.py

```diff
@@ -1,49 +1,70 @@
-# This is the ViUR default module importer;
-# If any other importing logic is wanted, please switch to manual import calls in this file, and remove
-# the dynamic code provided below.
+"""
+This is the ViUR default module importer.
 
-import logging
-import os
-import viur
+If any other importing logic is wanted, please switch to manual import calls in this file, and remove
+the dynamic code provided below.
+"""
 
 ####################################
 # Automatic imports are done here! #
 ####################################
 
-# start of script
+import logging
+import types
+import viur
+from pathlib import Path
+
 _viur_modules = {}
 
-for _module in os.listdir(os.path.dirname(__file__)):
+BLACKLIST = []  # filenames that should be blacklisted for the import
+
 
-    if _module == "__init__.py" or not _module.endswith(".py"):
-        continue
+def _import_modules(_path, _prefix=""):
+    for _module in Path(_path).iterdir():
```

Comment:
It's already a `Path` instance.
```suggestion
    for _module in _path.iterdir():
```

Reply:
this doesnt work unfortunately, i had it like this in my first attempt to implement pathlib